futures/src/lib.rs

```diff
@@ -58,7 +58,14 @@ compile_error!("The `never-type` feature requires the `nightly` feature as an ex
 #[cfg(feature = "std")]
 #[doc(hidden)] pub use futures_io::{AsyncRead, AsyncWrite, AsyncSeek, AsyncBufRead};
 #[cfg(feature = "std")]
-#[doc(hidden)] pub use futures_util::{AsyncReadExt, AsyncWriteExt, AsyncSeekExt, AsyncBufReadExt};
```

Comment:
Please put this back and write `#[doc(hidden)] pub use futures_util::OptionExt;` on a new line (this line is only compiled if the `std` feature is enabled).

Reply:
Ah, good catch. Thanks.